ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/components/style/StackComponentStyle.kt

```diff
@@ -0,0 +1,38 @@
+package com.revenuecat.purchases.ui.revenuecatui.components.style
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.runtime.Immutable
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.unit.Dp
+import com.revenuecat.purchases.paywalls.components.properties.Dimension
+import com.revenuecat.purchases.paywalls.components.properties.Size
+import com.revenuecat.purchases.ui.revenuecatui.components.properties.BackgroundStyle
+import com.revenuecat.purchases.ui.revenuecatui.components.properties.BorderStyle
+import com.revenuecat.purchases.ui.revenuecatui.components.properties.ShadowStyle
+
+@Suppress("LongParameterList")
+@Immutable
+internal class StackComponentStyle(
+    @get:JvmSynthetic
```

Comment:
Mentioned this in another PR but same here. I think we could remove these synthetic annotations for internal code, it would be more readable as well IMO. Wdyt?